contessa/alembic/versions/480e6618700d_change_type_to_name.py

```diff
@@ -0,0 +1,66 @@
+"""change type to name
+
+Revision ID: 480e6618700d
+Revises: 54f8985b0ee5
+Create Date: 2019-11-04 14:12:21.483875
+
+"""
+from alembic import op
+from alembic import context
+from sqlalchemy import create_engine
+import sqlalchemy as sa
+from sqlalchemy import inspect
+
+
+# revision identifiers, used by Alembic.
+revision = "480e6618700d"
+down_revision = "54f8985b0ee5"
+branch_labels = None
+depends_on = None
+
+config = context.config
+url = config.get_main_option("sqlalchemy.url")
+schema = config.get_main_option("schema")
+table_prefix = "quality_check"
+
+
+def get_quality_tables():
+    engine = create_engine(url)
+    inspector = inspect(engine)
+
+    all_tables = inspector.get_table_names(schema=schema)
+    quality_tables = [x for x in all_tables if x.startswith(table_prefix)]
```

Comment:
hmm, we allow to change names for users. I wouldn't rely on this convention of `table_prefix` then. I can think of 2 solutions: 

1. Can we do it for all tables? in case of error just skip and log at the end. It should be configurable `fail-fast/fail-silently mode` for user in CLI (can be done later). Not sure if doable tho, cause dunno if e.g. `op.alter_column` plans the queries or also executes them, so not sure if we can fail silently.

2. We will go with `--table-prefix` option. if there's no prefix passed, we'll use the default one (it should be as a CONSTANT taken from Contessa repo btw). I would add option to pass list of tables (file, str, stdout) so people can define it themselves if they have other names and pass it in simple bash/python scirpt, e.g. `python3 get_my_tables.py | contessa migration --etc --etc` while get my tables would connect to the database and retrieve tables by custom rules

Reply:
I see your points. The reason why I used this design is, that in the data quality schema user could have more tables (no only contessa tables). Option 1. will not work, because some migration can succeed on tables (e.g. add column, rename column if column exists) that are not data quality tables.

Option 2. was something I planned to do. Supporting the list of tables I would move to a separate task. Do you agree?

Reply:
okey, lets do it in another PR once we have e2e solution. just keep in mind that without it it can be unusable, so its should be done right after.

Reply:
In our case supporting only prefix is usable. For some special cases, we could add the possibility to list only tables you'd like to migrate but this is more complex - not implementation but logic consequences. If you will do migration of only subset of tables you do not have information about which table is in which version. The prefix is no universal solution as well because you do not have such info as well, but it is more strict like tables listing. The most appropriate solution is maybe hardcoded schema and prefix. Because of these open issues I'd like to split it to another feature.

Reply:
we can combine both options :) prefix (default or configurable) + all custom tables if any is listed

anyway, yes, lets keep it to another PR

Reply:
added as separate issue